### PR TITLE
chore(web): more translations for user settings and admin pages

### PR DESCRIPTION
**Pull request by michelheusschen**
There are still a few translations missing that include HTML within the message. I'm still trying to figure out the best approach for those.

### CODE REVIEW

File: web/src/lib/components/admin-page/jobs/job-tile.svelte [hunk 1]
```diff
@@ -119,12 +119,14 @@
         color="light-gray"
         on:click={() => dispatch('command', { command: JobCommand.Start, force: false })}
       >
-        <Icon path={mdiAlertCircle} size="36" /> DISABLED
+        <Icon path={mdiAlertCircle} size="36" />
+        {$t('admin.disabled').toUpperCase()}
```
Comment (danieldietzler) [inline web/src/lib/components/admin-page/jobs/job-tile.svelte:123]: Should disabled be a general thing? I could imagine that being used in other non-admin places as well...?

File: web/src/lib/components/admin-page/settings/admin-settings.svelte [hunk 2]
```diff
@@ -53,7 +53,7 @@
     }
 
     notificationController.show({
-      message: 'Reset settings to the recent saved settings',
+      message: $t('reset_settings_to_recent_saved'),
```
Comment (danieldietzler) [inline web/src/lib/components/admin-page/settings/admin-settings.svelte:56]: This should be in under admin I believe

File: web/src/lib/components/user-settings-page/partner-settings.svelte [hunk 3]
```diff
@@ -151,22 +151,28 @@
...
-            <p class="text-md">{partner.user.name} can access</p>
+            <p class="text-xs font-medium my-4">
+              {$t('shared_with_partner', { values: { partner: partner.user.name } }).toUpperCase()}
+            </p>
+            <p class="text-md">{$t('partner_can_access', { values: { partner: partner.user.name } })}</p>
             <ul class="text-sm">
               <li class="flex gap-2 place-items-center py-1 mt-2">
-                <Icon path={mdiCheck} /> All your photos and videos except those in Archived and Deleted
+                <Icon path={mdiCheck} />
+                {$t('partner_can_access_photos')}
```
Comment (danieldietzler) [inline web/src/lib/components/user-settings-page/partner-settings.svelte:161]: Maybe `partner_can_access_assets` makes more sense here, as it also includes videos? 🤔

File: web/src/lib/utils.ts [hunk 4]
```diff
@@ -115,26 +117,28 @@ export const downloadRequest = <TBody = unknown>(options: DownloadRequestOptions
...
-    [JobName.FaceDetection]: 'Face Detection',
-    [JobName.FacialRecognition]: 'Facial Recognition',
-    [JobName.VideoConversion]: 'Transcode Videos',
-    [JobName.StorageTemplateMigration]: 'Storage Template Migration',
-    [JobName.Migration]: 'Migration',
-    [JobName.BackgroundTask]: 'Background Tasks',
-    [JobName.Search]: 'Search',
-    [JobName.Library]: 'Library',
-    [JobName.Notifications]: 'Notifications',
+export const getJobName = derived(t, ($t) => {
```
Comment (danieldietzler) [inline web/src/lib/utils.ts:120]: Fancy!

File: web/src/lib/utils.ts [hunk 5]
```diff
@@ -222,11 +226,13 @@ export const getAssetJobIcon = (job: AssetJobName) => {
...
 export const copyToClipboard = async (secret: string) => {
+  const $t = get(t);
+
   try {
     await navigator.clipboard.writeText(secret);
-    notificationController.show({ message: 'Copied to clipboard!', type: NotificationType.Info });
+    notificationController.show({ message: $t('copied_to_clipboard'), type: NotificationType.Info });
   } catch (error) {
-    handleError(error, 'Cannot copy to clipboard, make sure you are accessing the page through https');
+    handleError(error, $t('cannot_copy_to_clipboard'));
```
Comment (danieldietzler) [inline web/src/lib/utils.ts:235]: Should this be in `errors.`?

File: web/src/routes/admin/repair/+page.svelte [hunk 6]
```diff
@@ -118,7 +118,10 @@
     try {
       const matched = await loadAndMatch([filename]);
       if (matched) {
-        notificationController.show({ message: `Matched 1 item`, type: NotificationType.Info });
+        notificationController.show({
+          message: $t('admin.matched_items', { values: { count: 1 } }),
```
Comment (danieldietzler) [inline web/src/routes/admin/repair/+page.svelte:122]: Maybe prefix that with `repair_`? Could also think about prefixing everything here with `repair_` (where it makes sense). In the admin settings we also tried to prefix stuff that belongs to one section

File: web/src/routes/admin/system-settings/+page.svelte [hunk 7]
```diff
@@ -191,19 +191,19 @@
...
           <Icon path={mdiDownload} size="18" />
-          Export as JSON
+          {$t('export_as_json')}
         </div>
       </LinkButton>
       <LinkButton on:click={() => inputElement?.click()}>
         <div class="flex place-items-center gap-2 text-sm">
           <Icon path={mdiUpload} size="18" />
-          Import from JSON
+          {$t('import_as_json')}
```
Comment (danieldietzler) [inline web/src/routes/admin/system-settings/+page.svelte:206]: `import_from_json`

File: web/src/lib/components/admin-page/jobs/job-tile.svelte (hunk 1 above)
Comment (michelheusschen) [reply web/src/lib/components/admin-page/jobs/job-tile.svelte:123]: That already existed, but I will move it

File: web/src/lib/utils.ts [hunk 8]
```diff
@@ -222,11 +226,13 @@ export const getAssetJobIcon = (job: AssetJobName) => {
...
 export const copyToClipboard = async (secret: string) => {
+  const $t = get(t);
+
   try {
     await navigator.clipboard.writeText(secret);
-    notificationController.show({ message: 'Copied to clipboard!', type: NotificationType.Info });
+    notificationController.show({ message: $t('copied_to_clipboard'), type: NotificationType.Info });
   } catch (error) {
-    handleError(error, 'Cannot copy to clipboard, make sure you are accessing the page through https');
+    handleError(error, $t('errors.cannot_copy_to_clipboard'));
```
Comment (jrasm91) [inline web/src/lib/utils.ts:235]:
```suggestion
    handleError(error, $t('errors.unable_to_copy_to_clipboard'));
```

**Review by jrasm91 — APPROVED** [review]
:heart:

File: web/src/routes/admin/repair/+page.svelte [hunk 9]
```diff
@@ -137,12 +140,15 @@
         count += await loadAndMatch(filenames.slice(index, index + chunkSize));
       }
     } catch (error) {
-      handleError(error, $t('errors.unable_to_check_items'));
+      handleError(error, $t('errors.repair_unable_to_check_items'));
```
Comment (danieldietzler) [inline web/src/routes/admin/repair/+page.svelte:143]: I feel like this should be done with plurals but we don't have the count here :/

**Review by danieldietzler — APPROVED** [review]
Thank you very much! :)

File: web/src/routes/admin/repair/+page.svelte (hunk 9 above)
Comment (michelheusschen) [reply web/src/routes/admin/repair/+page.svelte]: Plurals don't seem right here, I've used the [{select} format](https://formatjs.io/docs/core-concepts/icu-syntax/#select-format) instead

Comment (michelheusschen) [reply web/src/routes/admin/repair/+page.svelte]: The `one` option can be omitted if there's no difference for a language

Comment (danieldietzler) [reply web/src/routes/admin/repair/+page.svelte]: Oh awesome!

**Review by zackpollard — APPROVED** [review]

**Pull request merged**